docs/content/Getting-Started/Core/04-Query-Cube-API.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Query Cube API
```

Comment:
Similar page in the "Cube Cloud" sequence is called "Query Cube". I would make their names same, e.g., "Query Data."